addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1510,22 +1515,7 @@ const Wysiwyg = Widget.extend({
         }
     },
     _editorOptions: function () {
-        var self = this;
-        var options = Object.assign({}, this.defaultOptions, this.options);
-        options.onChange = function (html, $editable) {
-            $editable.trigger('content_changed');
-            self.trigger_up('wysiwyg_change');
-        };
-        options.onUpload = function (attachments) {
-            self.trigger_up('wysiwyg_attachment', attachments);
-        };
-        options.onFocus = function () {
-            self.trigger_up('wysiwyg_focus');
-        };
-        options.onBlur = function () {
-            self.trigger_up('wysiwyg_blur');
-        };
-        return options;
+        return Object.assign({}, this.defaultOptions, this.options);
```

Comment:
Why this change in this commit ?

Reply:
Because this commit fixes the content_changed issue that broke checklists, and this is the dead code that was never triggered. It also includes attachment, focus and blur, which are already handled elsewhere, but I figured it would be a bit ridiculous to just remove content_changed here and create a separate commit for the other three, wouldn't it?